xwiki-commons-core/xwiki-commons-xml/src/main/java/org/htmlcleaner/XWikiDOMSerializer.java

```diff
@@ -131,7 +132,7 @@ private Document createDocument(DocumentBuilder builder, TagNode rootNode)
             // Fix any invalid attribute names
             //
             if (!props.isAllowInvalidAttributeNames()) {
-                attrName = Utils.sanitizeXmlAttributeName(attrName, props.getInvalidXmlAttributeNamePrefix());
+                attrName = Utils.sanitizeXmlIdentifier(attrName, props.getInvalidXmlAttributeNamePrefix());
```

Comment:
This probably needs a comment explaining why the dedicated ``sanitizeXmlAttributeName`` is not used.

Reply:
it's actually a rename inside HtmlCleaner: the method `Utils.sanitizeXmlAttributeName` does not exist anymore.

Reply:
OK.